app/assets/config/explore.ts

```diff
@@ -222,6 +222,6 @@ export const DEFAULT_EXPLORE_RULES = {
   edgeDirection: 'outgoing',
   stepsType: 'single',
   step: 1,
-  vertexColor: 'groupByTag',
+  vertexSets: 'groupByTag',
```

Comment:
vertexSets may be not a good name, I don't know what's the meaning of it

Reply:
how about vertexStyle? In addition, `groupByTag` can be more detailed to `colorGroupByTag`